velox/vector/ComplexVector.cpp

```diff
@@ -642,7 +663,7 @@ uint64_t MapVector::hashValueAt(vector_size_t index) const {
   }
   auto offset = rawOffsets_[index];
   auto size = rawSizes_[index];
-  // hashMix is commutative, thus we do not canonicalize first.
+  // hashMix is commutative, thus we do not sort first.
```

Comment:
Is this comment accurate? If hashMix is commutative, then why is it replaced with folly::hash::commutative_hash_combine above?